strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml

```diff
@@ -0,0 +1,17 @@
+<databaseChangeLog xmlns:o="http://www.unbroken-dome.org/schema/liquibase-orientdb"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
+
+    <changeSet id="v1.0.0.8" author="pablotr87@gmail.com">
+
+        <o:dropProperty name="AuthorizationConfig.roles" className="AuthorizationConfig" propertyName="roles" />
```

Comment:
@ptirador I think sometiong wrong here :)
probably it should be:
```
<o:dropProperty name="roles" className="AuthorizationConfig"  />
```

Reply:
What's the problem?
If I change it to that, an error occurs:
```
Caused by: liquibase.exception.ValidationFailedException: 
Validation Failed:
     1 changes have validation failures
          propertyName: must not be null, classpath:/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml::v1.0.0.8
```

Reply:
> Which one's the correct one?

CASE 4

I have also checked and it works like this:
```
        <o:dropProperty className="AuthorizationConfig" propertyName="roles" />
        <o:dropClass name="Roles" />
        <o:createProperty name="roles" type="embeddedset" linkedType="Role" className="AuthorizationConfig" />

```